arnold/models/database/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .sample import Sample
+from .lims_sample import LimsSample
 from .step import Step
```

Comment:
should we use absolute imports in here as well as in the other repos?

Reply:
fixed